desktop/src/main/java/bisq/desktop/main/support/dispute/DisputeChatPopup.java

```diff
@@ -96,7 +96,7 @@ public void openChat(Dispute selectedDispute, DisputeSession concreteDisputeSess
         AnchorPane.setRightAnchor(chatView, 10d);
         AnchorPane.setTopAnchor(chatView, -20d);
         AnchorPane.setBottomAnchor(chatView, 10d);
-
+        pane.setStyle("-fx-background-color: -bs-color-blue-5");
```

Comment:
Didn't it work to use a class in that case?

Reply:
Thanks for the suggestion, done! :+1: